kr8s/_api.py

```diff
@@ -663,6 +663,14 @@ async def async_api_versions(self) -> AsyncGenerator[str]:
             for version in group["versions"]:
                 yield version["groupVersion"]
 
+    async def async_create(self, resources: list[APIObject]):
+        async with asyncio.TaskGroup() as tg:
```

Comment:
The `asyncio.TaskGroup()` was introduced in Python 3.11, so we need to use the `anyio.create_task_group()` polyfill for now to maintain support for older Python versions.

Reply:
Right. I switched to `anyio.create_task_group()` and checked it works as expected.